### PR TITLE
Check for race condition when getting services

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/sources/OSGiConfigUtils.java [hunk 1]
```diff
@@ -146,6 +146,7 @@ private static ConfigVariables getConfigVariables(BundleContext bundleContext) t
      * @throws InvalidFrameworkStateException if the server OSGi framework is being shutdown
      * @throws ServiceNotFoundException       if an instance of the requested service can not be found
      */
+    @FFDCIgnore(InvalidFrameworkStateException.class)
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/sources/OSGiConfigUtils.java]: why is this needed?

Comment (benjamin-confino) [reply dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/sources/OSGiConfigUtils.java]: In all honesty I'm going by the fact it's on a lot of other methods. Though it seems reasonable that we don't want to throw an exception that only occurs during shutdown as it's unlikely to be an issue the customer needs to worry about.

Comment (tevans78) [reply dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/sources/OSGiConfigUtils.java]: i think this will fail to build ... the build checks that all FFDCIgnore exceptions are actually thrown

Comment (tevans78) [reply dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/sources/OSGiConfigUtils.java]: I think this will fail to build ... the build checks that all FFDCIgnore exceptions are _caught_ in the method.

**Comment by benjamin-confino** [comment]
#build
